wandb/integration/tensorboard/monkeypatch.py

```diff
@@ -122,23 +129,27 @@ def new_csfw_func(*args, **kwargs):
     wandb.patched["tensorboard"].append([module, "create_summary_file_writer"])
 
 
-def _patch_file_writer(writer, module, save=None, root_logdir=None):
+def _patch_file_writer(
+    writer: Any, module: Any, save: bool = True, root_logdir: str = "",
+) -> None:
     # This configures non-TensorFlow Tensorboard logging, or tensorflow <= 1.15
     old_efw_class = writer.EventFileWriter
```

Comment:
since this is a class, maybe we should rename it to match python conventions for naming classes?